vignettes/how-to-update-released-site.Rmd

```diff
@@ -14,64 +14,90 @@ knitr::opts_chunk$set(
 )
 ```
 
-## Checklist
-
-If you're already up-to-speed, here's the checklist:
-
-* Make sure your dev site looks the way you want before embarking on an update
-  of your released site. Here are good instructions for updating an existing
-  site for pkgdown 2.0.0, Bootstrap 5, and "tidyverse/tidytemplate" (only
-  applies to certain packages):  
-  <https://tidytemplate.tidyverse.org>
-* Make sure you are using automatic development mode. In `_pkgdown.yml`:
-
-  ``` yaml
-  development:
-    mode: auto
-  ```
-* Push/pull to sync your default branch.
-* Create and checkout a branch based on the tag of your most recent GitHub/CRAN
-  release.
-  
-  Example for readr, whose latest release is 2.1.1:  
-  `git checkout -b update-pkgdown-2-1-1 v2.1.1`
-  
-  General pattern:  
-  `git checkout -b NEW-BRANCH-NAME NAME-OF-RELEASE-TAG`
-* Backport innovations from the future. Example of a useful opening move:  
-  `git checkout main -- _pkgdown.yml .github/workflows/pkgdown.yaml`
-
-  Most likely candidates:
-  - `_pkgdown.yml`
-  - `.github/workflows/pkgdown.yaml`
-  - `Config/Needs/website: tidyverse/tidytemplate` in DESCRIPTION
-  - Any fixes to `README.Rmd` / `README.md` or released roxygen comments. Don't
-    forget to `devtools::build_readme()` or `devtools::document()`, in that
-    case.
-  
-  These are the two main Git commands you'll need:
-  ```
-  git checkout REF -- path/to/a/specific/file
-  git cherry-pick SHA
-  ```
-* Monitor your progress locally with `pkgdown::build_site()`.
-* Push this branch, maybe using `usethis::pr_push()`. Don't bother opening
-  a pull request (the branch will still be created and pushed).
-* Go to your package's GHA page, maybe using `usethis::browse_github_actions()`.
-  Select the pkgdown workflow. Click *Run workflow* and select the branch you
-  just pushed. If there's no dropdown menu for this, that means your pkgdown
-  workflow config is not current.
-* Scrutinize your newly published **released** site and make sure things look
-  right.
-* You can let this branch sit around for a while, in case you need to iterate
-  on this or if you'd like to backport more niceties before your next CRAN
-  release. Or you can delete immediately, if you're happy with the released
-  site.
+This vignette shows you how to update the released version of your site to match the dev version of the site, so the first step is to ensure that the dev site looks the way that you want it.
+
+## Process
+
+If you're up to speed with the basic idea and just want some code to follow here it us.
+Otherwise, read more below.
+
+### Setup
+
+First, make sure you're in the main branch, and you have the latest version:
```

Comment:
```suggestion
First, make sure you're in the `main` branch, and you have the latest version:
```

FWIW `usethis::pr_merge_main()` is another reasonable thing to use here, but I get that maybe you prefer to use gert.

Reply:
Yeah, just feels weird to use a "PR" function here